temba/channels/models.py

```diff
@@ -2013,10 +2013,8 @@ def send_vumi_message(cls, channel, msg, text):
         is_ussd = channel.channel_type in Channel.USSD_CHANNELS
         channel.config['transport_name'] = 'ussd_transport' if is_ussd else 'mtech_ng_smpp_transport'
 
-        if msg.response_to_id:
-            in_reply_to = Msg.objects.get(pk=msg.response_to_id).external_id
-        else:
-            in_reply_to = None
+        in_reply_to = Msg.objects.values_list(
+            'external_id', flat=True).get(pk=msg.response_to_id)
```

Comment:
@nicpottier ah my bad then, I must have used it wrong when I tried then. I wonder if it has the same performance as only or defer. Sorry if I created confusion, I just felt the need to comment, because I'm working on something very similar.